Tasks/AzurePowerShellV5/InitializeAz.ps1

```diff
@@ -97,8 +97,8 @@ if ($endpointObject.scheme -eq 'ServicePrincipal') {
         $TenantId = $endpointObject.tenantId
         $additional = @{ TenantId = $TenantId }
 
-        Write-Host "##[command] Set-AzContext -SubscriptionId $SubscriptionId $(Format-Splat $additional)"
-        $null = Set-AzContext -SubscriptionId $SubscriptionId @additional
+        Write-Host "##[command] Set-AzContext -SubscriptionId $SubscriptionId @processScope $(Format-Splat $additional)"
+        $null = Set-AzContext -SubscriptionId $SubscriptionId @processScope @additional
```

Comment:
Is this change not required for set-azurermcontext ?

Reply:
Doing it for az now, will fix azurerm later as the bug that I have to fix in August patch on TFS2019 , requires az to be fixed for now